File: app/src/main/java/com/example/watering/investrecord/CategoryMainDBHelper.java
package com.example.watering.investrecord;

import android.content.Context;

/**
 * Created by watering on 17. 10. 21.
 */

@SuppressWarnings("DefaultFileTemplate")
class CategoryMainDBHelper extends DBHelper {

    public CategoryMainDBHelper(Context context) {
        super(context);

        TABLE_NAME = "tbl_category_main";
        COLUMNS = new String [] {"name TEXT UNIQUE","kind TEXT"};
    }
}

File: app/src/main/java/com/example/watering/investrecord/DBHelper/CategoryMainDBHelper.kt
package com.example.watering.investrecord.DBHelper

import android.content.Context

/**
 * Created by watering on 17. 10. 21.
 */

class CategoryMainDBHelper(context: Context) : DBHelper(context) {

    init {
        TABLE_NAME = "tbl_category_main"
        COLUMNS = arrayOf("name TEXT UNIQUE", "kind TEXT")
    }
}
